playbooks/common/openshift-cluster/upgrades/upgrade.yml

```diff
@@ -203,7 +215,7 @@
     command: >
       {{ openshift.common.admin_binary }} manage-node {{ openshift.common.hostname | lower }} --schedulable=true
     delegate_to: "{{ groups.oo_first_master.0 }}"
-    when: inventory_hostname in groups.oo_nodes_to_config and openshift.node.schedulable | bool
```

Comment:
Doesn't appear to be that way, I have no openshift_schedulable set in inventory on any hosts and I'm getting debug's on openshift.node.schedulable coming out correct.